### ISSUE TITLE
How to mapping ui query with oal

### ISSUE THREAD

**Issue by EvanLjp** [apache/skywalking#4592]
Please answer these questions before submitting your issue.

- Why do you submit this issue?
- [ ] Question or discussion

___
### Question
- What do you want to know?
the purpose of  oal expression is to compute metric by a simple way.
In my mind,the using of it contains the 2 following ways:
1. for alert: 
    so the alerm-settings.xml metrcs-nem is mapping to the official_analysis.oal
2. for watching:
     the ui dashborad

**1st querstion:** i don't know the mapping between rules and interfaces.
 for example:
the name of ServiceReponseTIme in oal is service_resp_time,however the name of ServiceReponseTIme in ui is serviceResponseTime.

![image](https://user-images.githubusercontent.com/31562192/77909943-c73a5e80-72c0-11ea-8a85-a39e64ed5113.png)
![image](https://user-images.githubusercontent.com/31562192/77909961-d0c3c680-72c0-11ea-9052-a31f2f8f3237.png)
**2nd question**:  Whether the ui can change dynamically follows the oal rules, 
How to do it if skywalking-ui have this ability ?

Thanks for your helping.

**Comment by wu-sheng** [comment]
> 1st querstion: i don't know the mapping between rules and interfaces.
for example:
the name of ServiceReponseTIme in oal is service_resp_time,however the name of ServiceReponseTIme in ui is serviceResponseTime.

This is normal case. UI is for human reading, and OAL is for developers. They are different.

> 2nd question: Whether the ui can change dynamically follows the oal rules,
How to do it if skywalking-ui have this ability ?

This is a complex feature. @TinyAllen used to try this, but don't work very well. So, currently, it is not there. UI doesn't support custom metrics.

**Comment by EvanLjp** [comment]
1. But how does the ui  and oal correspond one by one?I found the name is different,is there a doc for explain it,or say all in the official oal.
2. the second question ,I found the past [issue]( https://github.com/apache/skywalking/issues/2053) which says ui support the custom metrics,but in your response I think it is unrealistic.
So i found the exporter module, is it a good way to get this custom information? How to use it?why don't combine it to promethus module. thanks for your helping


so the metrics exporter is a way to get this custom
